src/main/java/net/dv8tion/jda/api/entities/Member.java

```diff
@@ -114,6 +114,26 @@
     @Nullable
     OffsetDateTime getTimeBoosted();
 
+    /**
+     * The time until which this Member is in time out.
+     * <br>If this Member is not in time out, this returns {@code null}.
+     *
+     * @return The time until which this Member is in time out or {@code null} if not in time out
+     */
+    @Nullable
+    OffsetDateTime getTimeUntilTimedOut();
+
+    /**
+     * Whether this Member is in time out.
+     * <br>While a Member is in time out, they cannot reply, send messages, react and speak in voice channels.
+     *
+     * @return True, if this Member is in time out
+     */
+    default boolean isTimedOut()
+    {
+        return getTimeUntilTimedOut() != null;
+    }
```

Comment:
Wouldn't it make sense to also check if this is from the past if it's not null?

Reply:
if the timeout expires the instance is updated because of the event